src/lib/geoip/altnames.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/naming-convention, quote-props */
+export const altNames: Record<string, string> = {
```

Comment:
The variable should be more descriptive; the file name is rather general and it could be assumed it contains aliases for more than just one field.

Reply:
How does cityAltnames sound? Or do we want to change the filename itself?

Reply:
I would name it “cities”, but “cityAltnames” works just fine too.